server/tests/src/test/java/org/infinispan/server/functional/RequestTracingIT.java

```diff
@@ -0,0 +1,122 @@
+ * in the main {@code lib} directory (as opposed to the {@code server/lib} directory used by this test).
+ *
+ * @since 10.0
+ */
+public class RequestTracingIT {
+   public static final String SERVICE_NAME = RequestTracingIT.class.getName();
+   public static final int NUM_KEYS = 10;
+   private static JaegerAllInOne JAEGER = new JaegerAllInOne("jaegertracing/all-in-one:latest");
+
+   @ClassRule
+   public static final InfinispanServerRule SERVER =
+         InfinispanServerRuleBuilder.config("configuration/ClusteredServerTest.xml")
+                                    .runMode(ServerRunMode.CONTAINER)
+                                    .numServers(1)
+                                    .mavenArtifacts("io.jaegertracing:jaeger-core:0.34.3",
+                                                    "io.jaegertracing:jaeger-thrift:0.34.3",
+                                                    "io.jaegertracing:jaeger-tracerresolver:0.34.3",
+                                                    "io.opentracing.contrib:opentracing-tracerresolver:0.1.5",
+                                                    "org.slf4j:slf4j-api:1.7.25",
+                                                    "com.squareup.okhttp3:okhttp:3.14.4",
+                                                    "com.squareup.okio:okio:1.13.0",
+                                                    "org.apache.thrift:libthrift:0.13.0",
+                                                    "io.opentracing:opentracing-api:0.31.0",
+                                                    "io.opentracing:opentracing-util:0.31.0",
+                                                    "io.opentracing:opentracing-noop:0.31.0",
+                                                    "org.slf4j:slf4j-jdk14:1.7.5")
+//                                    .mavenArtifacts("io.jaegertracing:jaeger-core:1.5.0",
+//                                                    "io.jaegertracing:jaeger-thrift:1.5.0",
+//                                                    "io.jaegertracing:jaeger-tracerresolver:1.5.0",
+//                                                    "io.opentracing.contrib:opentracing-tracerresolver:0.1.8",
+//                                                    "org.slf4j:slf4j-api:1.7.28",
+//                                                    "com.squareup.okhttp3:okhttp:4.9.0",
+//                                                    "com.squareup.okio:okio:2.8.0",
+//                                                    "org.apache.thrift:libthrift:0.13.0",
+//                                                    "io.opentracing:opentracing-api:0.33.0",
+//                                                    "io.opentracing:opentracing-util:0.33.0",
+//                                                    "io.opentracing:opentracing-noop:0.33.0",
+//                                                    "org.slf4j:slf4j-jdk14:1.7.5")
```

Comment:
Remove comments? Or is this needed by QE?

Reply:
I wanted to make it easy to test with both opentracing 0.31.0 and 0.33.0, although changing the version in the parent POM is still needed for now. Ideally I want the test to run with both impls in the same build.